storybook/webpack.config.js

```diff
@@ -1,26 +1,13 @@
-// you can use this file to add your custom webpack plugins, loaders and anything you like.
-// This is just the basic way to add addional webpack configurations.
-// For more information refer the docs: https://goo.gl/qPbSyX
+// load the default config generator.
+const genDefaultConfig = require('@kadira/storybook/dist/server/config/defaults/webpack.config.js')
```

Comment:
We shouldn't add the default configurations here, you should be able resolve aliases without default storybook config